htlcswitch/link.go

```diff
@@ -1953,6 +1965,8 @@ func (l *channelLink) updateCommitTx() error {
 		return err
 	}
 
+	l.cfg.PendingCommitTicker.Pause()
```

Comment:
Shouldn't this instead unpause (though this calls pause?) after we _receive_ their revoke message.

Reply:
In some cases, we'll respond with a commit sig after each revoke message, but if we don't owe them any updates the receipt of their revoke message signals the completion of the pending commitment dance.

Reply:
i think the way it is is correct. we start the timer (`Resume()`) when we already have a pending remote commitment _and_ we have more updates that couldn't be added because the revocation window was exhausted. If all pending updates had been covered in the pending remote commitment, there's no need to start the timer since we can't cancel anything that's locked in.

wrt to putting the `Pause()` after we receive the revoke and ack, that would also work. i think this is safer tho, since it actually cancels the ticker once the pending packets are actually covered by a commitment, rather than assuming that just because we received the revocation that the pending updates will be covered by the next signature.

Reply:
Makes sense, I also initially missed that we only start the ticker if we're unable to sign, meaning the other party owes us a revocation.